partshop-backend/src/main/java/hu/kosztaauto/partshop/controller/RequestController.java

```diff
@@ -21,13 +23,23 @@ public class RequestController {
     @Autowired
     private RaktarService raktarService;
 
-    @GetMapping("/getAllParts")
-    public List<Alkatresz> getAllParts() {
-        return alkatreszService.getAllParts();
+    @GetMapping("/getAllItems")
```

Comment:
You should keep your uris lowercase.

Reply:
I prefer them camel, but will make a fix for namings in next version